components/feature/media/src/test/java/mozilla/components/feature/media/fullscreen/MediaSessionFullscreenFeatureTest.kt

```diff
@@ -320,4 +324,50 @@ class MediaSessionFullscreenFeatureTest {
 
         assertNotEquals(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE, externalActivity.requestedOrientation)
     }
+    @Suppress("Deprecation")
```

Comment:
Just a drive-by question: What is the deprecation suppression for?

Reply:
Thank you for pointing that out !  It's not necessary for this test case. I copied a part of my previous test case and forgot to remove the deprecation line. Will update the code.